CI/espresso_tests/integration_tests/test_rl_script.py

```diff
@@ -119,7 +119,7 @@ def __call__(self, x):
         network = srl.networks.FlaxModel(
             flax_model=acto_critic,
             optimizer=optax.adam(learning_rate=0.001),
-            input_shape=(1,),
+            input_shape=(1, 1),
```

Comment:
Do we actually have to specify the shape like that?

Reply:
no, that should be gone actually, the one above is correc.t

Reply:
Oh but this was fixed in the next pr... I just had this in here to see if that was the problem.